docs/experiment/experiment.rst

```diff
@@ -0,0 +1,81 @@
+Experiment
+==========
+
+The main component of an experiment in :py:mod:`EasyReflectometry` is the `model`. 
+This is a description of the `sample` and the environment in which the experiment is performed. 
+The `Model` is used to calculate the reflectivity of the `sample` at a given set of angles (`q` values).
+The `resolution_functions` are used to account for non-ideal effects in the instrument.
+
+
+:py:class:`Model`
+-----------------
+
+A `Model` instance contains a `sample` and variables describing experimental settings.
+To be able to compute reflectivities it is also necessary to have a `calculator` (interface).
+
+.. code-block:: python 
+
+   from EasyReflectometry.calculators import CalculatorFactory
+   from EasyReflectometry.experiment import Model
+   from EasyReflectometry.sample import Sample
+
+   default_sample = Sample()
+   m = Model(
+      sample=default_sample,
+      scale=1.0,
+      background=1e-6
+   )
+
+   interface = CalculatorFactory()
+   model.interface = interface
+
+This will create a :py:class:`Model` instance with the `default_sample` and the environment variables `scale` factor set to 1.0 and a `background` of 1e-6.
+Following the `interface` is set to the default calculator.
+
+
+:py:mod:`resolution_functions`
+------------------------------
+A resolution function enables the `EasyReflectometry` model to account for various non ideal effects that might be present in the experimental setup.
+When determining reflectivity the resolution function defines the smearing to apply.
+For a given Q point such smearing is applied by determining an average of the neighboring Q point weigthed by a normal distribution, 
+which has a Q point dependent Full Width at the Half Maximum (FWHM) that again is defined by the resolution function.
+
+:py:func:`percentage_fhwm_resolution_function`
+Often we rely on a resolution function that has a simple functional dependecy of the Q point.
+By this is understood that the applied smearing in an Q point has a FWHM that is given as a percentage of the value of the Q point.
+
+.. code-block:: python 
+
+   from EasyReflectometry.experiment import Model
+   from EasyReflectometry.experiment import percentage_fhwm_resolution_function
+
+   resolution_function = percentage_fhwm_resolution_function(1)
```

Comment:
Maybe `1` --> `1.0` to make it more explicit what it is doing

Reply:
done